### DISCUSSION TITLE
exportImage in v7?

### DISCUSSION THREAD

**Discussion by hanna-becker** [katspaugh/wavesurfer.js#3088] [Q&A]
Hey there, can we still export the waveform as an image in v7?

In v6, this worked for me:
```
const waveformImageBlob = (await waveSurferInstance.exportImage('image/png', 1, 'blob'))[0];
```

In v7 the exportImage method is removed.

**Comment by katspaugh** [comment]
I missed that method, let's restore it.

**Reply by hanna-becker** [reply]
awesome!
